website/pages/en/versions.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
Is this Facebook's code ?
If not, remove the license note

Reply:
It's generated by docusaurus, should I remove this part?